app/assets/src/components/common/MetadataManualInput.jsx

```diff
@@ -80,6 +80,9 @@ class MetadataManualInput extends React.Component {
         )
       }
     });
+
+    // Default to the first host genome (Human)
+    this.props.samples.map(sample => this.updateHostGenome(1, sample));
```

Comment:
Looks like I had a bug in my code from before. `this.state.currentHostGenome` should be removed (and also `handleHostGenomeChange`). 

`<Dropdown value={this.state.currentHostGenome} />` should be changed to `<Dropdown value={sampleHostGenomeId} />`

As it is right now, "Human" will not show up in the host genome dropdowns at the start. They'll still be empty.

Reply:
👍